src/analyses/local_safe_pointers.cpp

```diff
@@ -82,7 +77,8 @@ get_conditional_checked_expr(const exprt &goto_guard)
 /// \param goto_program: program to analyse
 void local_safe_pointerst::operator()(const goto_programt &goto_program)
 {
-  std::set<exprt> checked_expressions;
+  std::set<exprt, base_type_comparet> checked_expressions(
+    base_type_comparet{ns});
```

Comment:
We don't usually use the `{ ... }` syntax.

Reply:
Yeah, but otherwise I need `(())` to disambiguate from a function declaration.